src/App/App.js

```diff
@@ -247,6 +248,19 @@ class App extends View {
 
   }
 
+  renderReconstructionsPage() {
+
+    const ReconstructionsPage = this.pages.get(`reconstructions`);
+    const reconstructionsPage = new ReconstructionsPage;
+
+    const oldPage = document.getElementById(`reconstructions-page`);
+    const newPage = reconstructionsPage.render();
+
+    if (oldPage) oldPage.replaceWith(newPage);
+    else this.nodes.wrapper.appendChild(newPage);
```

Comment:
As you noticed, it turns out we don't need this anymore!

```suggestion
    const el = reconstructionsPage.render();

    this.nodes.wrapper.appendChild(newPage);
```